atlasdb-perf/src/main/java/com/palantir/atlasdb/performance/backend/DockerizedDatabaseUri.java

```diff
@@ -33,11 +33,11 @@ public static DockerizedDatabaseUri fromUriString(String uri) {
         String[] parts = uri.trim().split(DELIMITER);
         String[] addrParts = parts[1].split(":");
         return new DockerizedDatabaseUri(
-                KeyValueServiceType.valueOf(parts[0]),
+                KeyValueServiceTypeInterface.keyValueServiceTypeFor(parts[0]),
                 InetSocketAddress.createUnresolved(addrParts[0], Integer.parseInt(addrParts[1])));
     }
 
-    public KeyValueServiceType getKeyValueServiceType() {
+    public KeyValueServiceTypeInterface getKeyValueServiceType() {
```

Comment:
Getter should match type name, or else describe what `type` is to the `DockerizedDatabaseUri`.